var/spack/repos/builtin/packages/py-lagom/package.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class PyLagom(PythonPackage):
+    """Lagom is a dependency injection container
+    designed to give you 'just enough' help with building your dependencies.
+    """
+
+    homepage = "https://lagom-di.readthedocs.io"
```

Comment:
If there's no source code on PyPI can you install from GitHub? https://github.com/meadsteve/lagom/tags has the URLs. Then we don't need all this Python version-specific complexity.

Reply:
done!